src/prefix.js

```diff
@@ -17,13 +17,19 @@ if (isBrowser) {
     ms: '-ms-',
     O: '-o-'
   }
+
+  // Order matters. We need to check Webkit the last one because
+  // other vendors use to add Webkit prefixes to some properties
+  const prefixes = ['Moz', 'ms', 'O', 'Webkit']
```

Comment:
The spec does not, however all implementations do ensure order, react relies on it as well. The entire jss json dsl does)